src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```

Comment:
```suggestion
export const MoviesList: React.FC<Props> = ({ movies }) => (
```
As you are using Typescript it will always be an array (empty or filled)